src/features/start/Start.tsx

```diff
@@ -60,6 +71,34 @@ const Start = () => {
           onChange={extractValueAndSet(setPlayerCount)}
         />
       </div>
+      <div className="row-span-1 col-span-1 my-2">
+        <label className="block text-gray-700 text-sm font-bold mb-2">
+          Time
+        </label>
+        <input
+          className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+          type="number"
+          placeholder="Players"
+          value={timePerPlayer}
+          min={0}
+          max={100000}
```

Comment:
this value should be set to 300 as we ask users to specify time on UI is seconds

Also lets update the server side code to ensure that no game can last for more than 5 minutes (300 seconds) - this is to avoid games to use up too much resources on the server side